planet/cli/orders.py

```diff
@@ -269,20 +296,39 @@ def read_file_json(ctx, param, value):
               help='Toolchain json file.',
               type=click.File('rb'),
               callback=read_file_json)
+@click.option('--email',
+              default=False,
+              is_flag=True,
+              help='Send email notification when Order is complete')
+@click.option(
```

Comment:
Is `--like` implemented here? I thought that got pushed out to post MVP to get this out earlier.